resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -245,4 +339,57 @@ function DoesRegistryKeyPropertyExist {
     $itemProperty = Get-ItemProperty -Path $Path  -Name $Name -ErrorAction SilentlyContinue
     return $null -ne $itemProperty
 }
+
+function Get-CombinedState {
+	param(
+		[Parameter(Mandatory)]
+		[BinarySettingState]$SmoothScrollListBoxes,
+		[BinarySettingState]$SlideOpenComboBoxes,
+		[BinarySettingState]$FadeOrSlideMenusIntoView,
+		[BinarySettingState]$ShowShadowsUnderMousePointer,
+		[BinarySettingState]$FadeOrSlideToolTipsIntoView,
+		[BinarySettingState]$FadeOutMenuItemsAfterClicking,
+		[BinarySettingState]$ShowShadowsUnderWindows,
+		[string] $AnimationEffectsProperty = 'UserPreferencesMask'
+	)
+
+	$OverallState = (Get-ItemPropertyValue -Path $global:AnimationEffectsRegistryPath -Name $AnimationEffectsProperty) | %{[System.Convert]::ToString($_,2).PadLeft(8,'0')}#Registry converts from hex to int, so this converts from int to binary.
+
+	$OverallState[0] = $OverallState[0][0..3]+$SmoothScrollListBoxes+$SlideOpenComboBoxes+$FadeOrSlideMenusIntoView+$OverallState[0][7] -join "" -replace "Enabled",1  -replace "Disabled",0 
+
+	$OverallState[1] = $OverallState[1][0..1]+$ShowShadowsUnderMousePointer+$OverallState[1][3]+$FadeOrSlideToolTipsIntoView+$FadeOutMenuItemsAfterClicking+$OverallState[1][6..7] -join "" -replace "Enabled",1  -replace "Disabled",0
+
+	$OverallState[2] = $OverallState[2][0..4]+$ShowShadowsUnderWindows+$OverallState[2][6..7] -join "" -replace "Enabled",1  -replace "Disabled",0
+	$OverallState = $OverallState | %{[convert]::ToInt32($_,2).ToString("X").PadLeft(2,'0')}#Converts from binary back to hex.

```

Comment:
I'm not the biggest fan of this conversion as its a bit confusing, I wonder if there's a better way to do this. Let me see if I can start a thread with some people to see what the best way to handle this would be

Reply:
I found another way. It's similar to the other function so hopefully it's more clear.